packages/create-svelte/templates/default/netlify.toml

```diff
@@ -1,3 +1,6 @@
 [build]
-  command = "npm run build"
+  command = "npx pnpm install --store=node_modules/.pnpm-store && npx pnpm build -r"
   publish = "build"
+
+[build.environment]
+  NPM_FLAGS = "--version"
```

Comment:
It'd be nice to have a little comment here explaining that this is a workaround to make the automatic running of npm a no-op.